tools/scripts/ManifestBuilder.rb

```diff
@@ -254,6 +201,30 @@ def get_animation_strings
     return strings
   end
 
+  def upload_localized_manifest(locale, strings)
```

Comment:
Maybe some documentation about what this method does and who should use it.
The name makes it sound like it is a generic localized manifest uploader, but it looks like it is specific to animation objects. Should the name be more specific? Or should the the method be made more generic?

Reply:
This ManifestBuilder is only for the animation library so I think the function name is fine. I want to clean up this file in general and I might consider changing the name of class in that.

I added some comments to this function.